docs/DeveloperGuide.md

```diff
@@ -371,9 +371,11 @@ _{More to be added}_
 
 ### Non-Functional Requirements
 
-1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
-2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+1.  (Compatibility) Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
+2.  (Capacity/Scalability) Should be able to hold up to 1000 students without a noticeable sluggishness in performance for typical usage.
```

Comment:
I feel that scalability is not an issue as our target user will never have that many students.

Reply:
That's true, maybe if we reduce it to just 100 instead of completely remove? Assuming a tutorial/lab class of maximum about 20 students like in CS2100, and that a TA could be teaching multiple lab sessions + another module, 100 might be a good upper limit? This means if we have a stress test in the future, 100 will be more than sufficient.

Reply:
Yups sounds good!